readme.md

```diff
@@ -84,52 +84,49 @@ your_method(){
 }
 ```
 
-# API Referenced
-
-> ## **setLanguage** ( lang ) : void
```

Comment:
I think we should to use that because it easy to view for someone who just want to scroll to view reference